scriptmodules/libretrocores/lr-mupen64plus-next.sh

```diff
@@ -20,18 +20,24 @@ function depends_lr-mupen64plus-next() {
     local depends=(flex bison libpng-dev)
     isPlatform "x11" && depends+=(libglew-dev libglu1-mesa-dev)
     isPlatform "x86" && depends+=(nasm)
-    isPlatform "rpi" && depends+=(libraspberrypi-dev)
+    isPlatform "videocore" && depends+=(libraspberrypi-dev)
+    isPlatform "mesa" && depends+=(libgles2-mesa-dev)
     getDepends "${depends[@]}"
 }
 
 function sources_lr-mupen64plus-next() {
     gitPullOrClone "$md_build" https://github.com/libretro/mupen64plus-libretro-nx.git GLideN64
+
+    # HACK: force EGL detection on FKMS
+    isPlatform "mesa" && applyPatch "$md_data/0001-force-egl.patch"
 }
 
 function build_lr-mupen64plus-next() {
     local params=()
-    if isPlatform "rpi"; then
+    if isPlatform "videocore"; then
         params+=(platform="$__platform")
+    elif isPlatform "videocore"; then
```

Comment:
I think this check should be for "mesa"?

Reply:
Well-spotted. The fallback defines at the end of the if block happened to be close enough to work for a GLES2 target, so the build still worked. It's fixed properly now, and to be sure it's OK I just tested a rebuild with success. Thanks.